count-pos-sum-neg.py

```diff
@@ -0,0 +1,15 @@
+def count_positives_sum_negatives(arr):
+    """From list, tally positive numbers, sum the negatives.""" 
+    answer_array = []
+    positives = 0
+    sum_negs = 0
+    if arr == []:
```

Comment:
ah, thanks. I'm starting to see different kinds of falsey-ness where I didn't before